src/main.vue

```diff
@@ -0,0 +1,418 @@
+<template>
+    <div id="kiwi-component-container">
+        <div id="kiwi-app"/>
+    </div>
+</template>
+<script>
+import _ from 'lodash';
+import Vue from 'vue';
+import i18next from 'i18next';
+import i18nextXHR from 'i18next-xhr-backend';
+import VueI18Next from '@panter/vue-i18next';
+
+import AvailableLocales from '@/res/locales/available.json';
+import FallbackLocale from '@/../static/locales/en-us.json';
+import App from '@/components/App';
+import StartupError from '@/components/StartupError';
+import Logger from '@/libs/Logger';
+import ConfigLoader from '@/libs/ConfigLoader';
+import state from '@/libs/state';
+import ThemeManager from '@/libs/ThemeManager';
+import InputHandler from '@/libs/InputHandler';
+import StatePersistence from '@/libs/StatePersistence';
+import * as Storage from '@/libs/storage/Local';
+import * as Misc from '@/helpers/Misc';
+import GlobalApi from '@/libs/GlobalApi';
+import { AudioManager } from '@/libs/AudioManager';
+import { SoundBleep } from '@/libs/SoundBleep';
+
+// Global utilities
+import '@/components/utils/TabbedView';
+import '@/components/utils/InputText';
+import '@/components/utils/IrcInput';
+import '@/components/utils/InputPrompt';
+import '@/components/utils/InputConfirm';
```

Comment:
Do these global components get added to the parent application? This could cause conflicts

Reply:
I think they do, but there's not really a way to prevent that. Perhaps a warning in the docs would suffice?